firstboot.d/01ipconfig

```diff
@@ -8,29 +8,56 @@
 [ "$IP_CONFIG" != "manual" ] && [ "$IP_CONFIG" != "static" ] && [ "$IP_CONFIG" != "dhcp" ] && exit 1
 [ ! -e /etc/network/interfaces ] && exit 1
 
+APP=$(turnkey-version | cut -d "-" -f 2)
```

Comment:
This will return unexpected results on appliances that have 2 (or more) part names. E.g. OTTOMH `domain-controller`, `tomcat-apache` & `mayan-edms`. IIRC there is a perl snippet floating around that is used elsewhere, or grep would also do the trick, e.g. `turnkey-version | grep -Eo '[0-9]+\.[0-9]+'`.

Reply:
@JedMeister - Good catch.  I think I stole that from somewhere.  Probably need to track it down and fix it at the source.

Reply:
@JedMeister - Your grep command line returns the appliance version and not the appliance name.  Since I'm only trying to match when the name is `lxc`, my version would continue to work until we define a new appliance named `lxc-something`.  Still it's best to fix the problem now and avoid a later surprise.

Reply:
It looks like this would work.
`turnkey-version | sed 's/turnkey-\(.*\)-[0-9]\+\.[0-9]\+-.*$/\1/'`